desc/protoprint/print.go

```diff
@@ -1674,11 +1676,15 @@ func (p *Printer) printOption(name string, optVal interface{}, w *writer, indent
 			fmt.Fprintf(w, "{ %s }", proto.CompactTextString(optVal))
 			return
 		}
-		m := proto.TextMarshaler{
-			Compact:   true,
-			ExpandAny: true,
+
+		m := &prototext.MarshalOptions{Multiline: false}
+
+		optText, err := m.Marshal(optVal.(protoV2.Message))
```

Comment:
I cant think of a case where `proto.Message` would ever not successfully cast to `protoV2.Message`

Reply:
That might be so, but this would just be cleaner to use `proto.MessageV2(optVal)` anyway. (https://pkg.go.dev/github.com/golang/protobuf/proto#MessageV2)

Reply:
Yep. Didnt realize that utility existed. Updated the PR to use it.